eng/Versions.props

```diff
@@ -247,10 +247,10 @@
     <PlaywrightSharpVersion>0.180.0</PlaywrightSharpVersion>
     <PollyExtensionsHttpVersion>3.0.0</PollyExtensionsHttpVersion>
     <PollyVersion>7.1.0</PollyVersion>
-    <SeleniumSupportVersion>4.0.0-alpha07</SeleniumSupportVersion>
+    <SeleniumSupportVersion>4.0.0-beta1</SeleniumSupportVersion>
     <SeleniumWebDriverMicrosoftDriverVersion>17.17134.0</SeleniumWebDriverMicrosoftDriverVersion>
     <SeleniumWebDriverChromeDriverVersion>87.0.4280.2000-beta</SeleniumWebDriverChromeDriverVersion>
```

Comment:
Current Chrome driver version is `89.0.4389.2300-beta`. Are we unable to update to match what we're likely downloading on the CI agents❔

Reply:
I was trying to remember how this version relates to https://github.com/dotnet/aspnetcore/blob/main/src/Shared/E2ETesting/selenium-config.json#L4. After looking through commits, I think it's safe to update this and then change the selenium-config.json to match it as needed.